ts/setup.ts

```diff
@@ -296,7 +296,8 @@ const settings = {
         "encryption": new Select(get("smtp-encryption")),
         "server": new Input(get("smtp-server")),
         "port": new Input(get("smtp-port")),
-        "password": new Input(get("smtp-password"))
+        "password": new Input(get("smtp-password")),
+        "cert-validation": new Input(get("smtp-cert-validation"))
```

Comment:
Not necessary as the corresponding element `smtp-cert-validation` doesn't exist in the setup html file (`html/setup.html`). This can be removed, as can the added comma on the line above.